src/tokens.d

```diff
@@ -160,11 +160,13 @@ enum TOK : int
     TOKimaginary64v,
     TOKimaginary80v,
 
+    // 123
```

Comment:
It doesn't depend on exact values. It's just there to make it easier to add missing values, which are given by index number.